src/less/widgets/bars.less

```diff
@@ -1,15 +1,19 @@
+/* Pushdown Styles */
+.pf-push-down {
+  transition: margin-top .3s;
+  margin-top: 0;
+
+  &.opened {
+    margin-top: 55px;
+  }
+}
+
 .pf-widget-bar {
-  background: @backgroundDefault;
-  color: @colorDefault;
   height: auto;
```

Comment:
we also need to adjust the z-index here to like 9999 or something super high. if tested against bootstrap its behind their default header styles (z-index 1030).